Makefile

```diff
@@ -4,7 +4,7 @@ VDIR=v5
 
 COV_OPTS=-coverprofile=coverage.txt -covermode=atomic
 
-all: testcov lint tidy
+all: testcov lint scan-gosec tidy
```

Comment:
I don't think we should include scan-gosec in the "all" target, to be honest.
Let's leave it as a separate target and then we can add a `make scan-gosec` command to the script section.  Would that work?

Reply:
Yep, that makes sense.